src/app/page.tsx

```diff
@@ -58,12 +74,12 @@ export default function Component() {
             </div>
           </div>
         </section>
-        <section className="w-full py-12 md:py-20 lg:pt-32 pb-24 bg-[#0E0E11] px-32">
+        <section className={`${theme === "true" ? "bg-[#0E0E11] text-gray-50" : "bg-[#ffffff] text-black"} w-full py-12 md:py-20 lg:pt-32 pb-24 px-32`}>
           <div className="container grid items-center justify-center gap-4 px-4 text-center md:px-6 lg:gap-10">
```

Comment:
instead of having this check in all the classes, can't we have a better way of doing this?

Reply:
I think, we have to make check in main divs, so that the classes adjust according to them.